src/test/resources/features/ReportAudit.feature

```diff
@@ -0,0 +1,9 @@
+Feature: reportAudit
+
+  Scenario: R0003 Verificar que se puede identificar los usuarios que realizaron modificaciones en datos del programa.
+    Given 'SSI' loaded
+    And User is authenticated in portal
+    And "salary" of the "Lavonne" employee is updated to "10000"
+    And click on "Reports" menu button
+    When click on "ReporteDeModificacionesDeDatos" submenu Button
```

Comment:
Let's add the page where you are executing the action. For example:  "And click on "Reports" menu button"  to  "And click on "Reports" menu button on 'Navigation Bar' top menu "